packages/reports/tests/unit/consumers/request/fili-test.ts

```diff
@@ -0,0 +1,159 @@
+      table: 'network',
+      filters: [
+        {
+          type: 'timeDimension',
+          field: 'network.dateTime',
+          parameters: { grain: 'day' },
+          operator: 'bet',
+          values: ['P1D', 'current'],
+          source: 'bardOne'
+        }
+      ],
+      sorts: [],
+      limit: null,
+      dataSource: 'bardOne',
+      requestVersion: '2.0',
+      columns: [
+        {
+          type: 'timeDimension',
+          field: 'network.dateTime',
+          parameters: { grain: 'day' },
+          source: 'bardOne'
+        },
+        {
+          type: 'metric',
+          field: 'adClicks',
+          parameters: {},
+          source: 'bardOne'
+        },
+        {
+          type: 'dimension',
+          field: 'age',
+          parameters: { field: 'id' },
+          source: 'bardOne'
+        }
+      ]
+    });
+
+    const modelFor = () => ({ request });
+
+    consumer.send(
+      RequestActions.UPDATE_COLUMN_FRAGMENT_WITH_PARAMS,
+      { modelFor },
+      request.nonTimeDimensions[0],
+      'field',
+      'desc'
+    );
+    assert.deepEqual(dispatchedActions, [], 'no actions are called when a dimension is updated');
+
+    consumer.send(
+      RequestActions.UPDATE_COLUMN_FRAGMENT_WITH_PARAMS,
+      { modelFor },
+      request.metricColumns[0],
+      'param',
+      'val'
+    );
+    assert.deepEqual(dispatchedActions, [], 'no actions are called when a metric is updated');
+
+    consumer.send(
+      RequestActions.UPDATE_COLUMN_FRAGMENT_WITH_PARAMS,
+      { modelFor },
+      request.timeGrainColumn,
+      'otherParam',
+      'week'
+    );
+    assert.deepEqual(
+      dispatchedActions,
+      [],
+      'no actions are called when the timeDimension updates a non-grain parameter'
+    );
+
+    consumer.send(
+      RequestActions.UPDATE_COLUMN_FRAGMENT_WITH_PARAMS,
+      { modelFor },
+      request.timeGrainColumn,
+      'grain',
+      'week'
+    );
+    assert.deepEqual(
+      dispatchedActions,
+      [RequestActions.UPDATE_FILTER],
+      'UPDATE_FILTER is dispatched when the timeDimension column and filter exists and the grain param is updated'
+    );
+    assert.deepEqual(
+      dispatchedActionArgs,
+      [request.dateTimeFilter, { parameters: { grain: 'week' } }],
+      'UPDATE_FILTER is dispatched with the updated grain parameter'
+    );
+
+    dispatchedActions.length = 0;
+    dispatchedActionArgs.length = 0;
+
+    config.navi.dataSources = [
+      ...config.navi.dataSources,
+      { name: 'Other Data Source', uri: 'data://stuff', type: 'elide' }
+    ];
+    request.dataSource = 'Other Data Source';
+
+    consumer.send(
+      RequestActions.UPDATE_COLUMN_FRAGMENT_WITH_PARAMS,
+      { modelFor },
+      request.timeGrainColumn,
+      'grain',
+      'week'
+    );
+    assert.deepEqual(dispatchedActions, [], 'no actions are called when the request is for an elide dataSource');
+  });
+});
```

Comment:
Do we already have acceptance tests that verify this behavior?

Reply:
I don't think we load any elide data sources into the reports package

Reply:
Should we update this test name to include `bard/fili`

https://github.com/yahoo/navi/blob/59189827f8c06a72166df1c1b38671b03d7fbc09/packages/reports/tests/acceptance/date-filter-test.js#L70